piton/ariane_build_tools.sh

```diff
@@ -88,7 +88,12 @@ else
   cd -
 
   cd build
-  ../configure --prefix=$ROOT/tmp/riscv-tests/build
+  tmp_dest=$PITON_ROOT/tmp
```

Comment:
I've been doing it for a while, but I don't like it so much, because then when I do "git status" I get that I made changed into "piton/design/chip/tile/ariane (untracked content)"

Reply:
I rather have it inside $PITON_ROOT/tmp, and add it to the gitignore

Reply:
Okay. Can I change it to $ARIANE_ROOT as you suggested in the beginning, instead?